cmd/vcert/enroll.go

```diff
@@ -112,7 +120,7 @@ func showEnrollmentUsage() {
 		"\t\tTPP & Cloud: trust_bundle, test_mode"))
 
 	fmt.Println("  -file")
-	fmt.Printf("\t%s\n", wrapArgumentDescriptionText("Use to specify a file name and a location where the resulting file should be written. If this option is used the key, certificate, and chain will be written to the same file. Example: /tmp/newcert.pem"))
+	fmt.Printf("\t%s\n", wrapArgumentDescriptionText("Use to specify a file name and a location where the resulting file should be written. If this option is used the key, certificate, and chain will be written to the same file. Do not use with -csr local. Example: /tmp/newcert.pem"))
```

Comment:
hi @mr-tron, sorry for not communicating all my changes.

we had a Venafi professional services rep onsite and while working with us and he pointed out that `-csr local` is not compatible with `-file`. If this is actually incorrect, or you would otherwise prefer me to remove, I can definitely do. I thought this was an opportunity to improve some of the command flag validation yet if this is incorrect, or there's a better comment to clarify the incompatibility, please let me know.

Reply:
`-file ` is a path for result certificate and chain. `-csr local` just specify method of creating csr.
So please remove these checks from pull request.

Reply:
sure